test/integration/test-amp-ad-3p.js

```diff
@@ -95,15 +95,26 @@ describe('amp-ad 3P', () => {
         });
         const {initialIntersection} = context;
         expect(initialIntersection.rootBounds).to.deep.equal(
-          layoutRectLtwh(0, 0, 500, 3000)
+          layoutRectLtwh(0, 0, window.innerWidth, window.innerHeight)
         );
         expect(initialIntersection.boundingClientRect).to.deep.equal(
           layoutRectLtwh(0, platform.isIos() ? 1001 : 1000, 300, 250)
         );
-        expect(initialIntersection.intersectionRect).to.deep.equal(
-          layoutRectLtwh(0, platform.isIos() ? 1001 : 1000, 300, 250)
-        );
-        expect(initialIntersection.intersectionRatio).to.equal(1);
+
+        // TODO: delete this assert. since depends on viewport.
```

Comment:
Let's clear this out.

Reply:
Yep! Wanted to discuss this first to see if it was ok to delete

Reply:
I think so. But we should probably ask some from ads to review.